### ISSUE TITLE
Minimap hovering only works in initial zone [5.1.0]

### ISSUE THREAD

**Issue by vaindil** [AeroScripts/QuestieDev#1494]
## Bug description
Changing zones causes Questie minimap hovering to stop working. Moving back to the original zone makes it work again, it seems that minimap hovering only works in the initial zone you were in when you loaded the interface. /reload causes the same symptoms, it resets the initial zone to your current zone.

## Questie version
5.1.0

**Comment by BreakBB** [comment]
@vaindil Could you explain/show how you're changing the minimap zone? :thinking:

**Comment by vaindil** [comment]
Sorry, I meant physically moving to a different zone. I was testing on the border of Blasted Lands and Swamp of Sorrows. I logged in in Blasted Lands, but after physically moving to Swamp of Sorrows I could no longer hover over minimap icons for Questie (there's a quest giver right on the border). Moving back to Blasted Lands allowed me to hover again. If I went into Swamp of Sorrows and did /reload, the same problem occurred in reverse.
